File: SmartMenza.Business/Models/NutritionGoals/DailyFoodIntakeResponse.cs
namespace SmartMenza.Business.Models
{
    public class DailyFoodIntakeResponse
    {
        public int DailyFoodIntakeId { get; set; }
        public DateTime Date { get; set; }

        public int DishId { get; set; }
        public string DishTitle { get; set; } = string.Empty;

        public int Calories { get; set; }
        public decimal Protein { get; set; }
        public decimal Fat { get; set; }
        public decimal Carbohydrates { get; set; }

        public string? ImgPath { get; set; }
    }
}

File: core/src/main/java/foi/cverglici/core/data/model/student/dailyfoodintake/DailyFoodIntakeResponse.kt
package foi.cverglici.core.data.model.student.dailyfoodintake

data class DailyFoodIntakeResponse(
    val dailyFoodIntakeId: Int,
    val date: String,
    val dishId: Int,
    val dishTitle: String,
    val calories: Int,
    val protein: Double,
    val fat: Double,
    val carbohydrates: Double,
    val imgPath: String?
)